GOFevaluation/utils.py

```diff
@@ -136,7 +135,29 @@ def _get_count_density(ns, be_first, be_second, data_sample):
     return ns
 
 
-def _get_equiprobable_binning(reference_sample, n_partitions, order=None):
+def _weighted_equi(n_bins, data, weights):
```

Comment:
This function is indirectly already covered by unittests but if you could add one or two dedicated ones for it, it would be amazing!! 😊 
Especially testing the correct behavior for non-unitary weights would be necessary, I think.

Reply:
(a sensible test could also be that the result with weights=None and weights=np.ones(...) gives the exact same result)